### PR TITLE
Update queries for slaves

### PR DESCRIPTION
**Pull request by CarsonF**
Replaced prepare() queries with executeQuery() since that supports read slaves. 
Replaced prepare() with query() on write queries because why do 3 method calls when you can do one?
Cleaned up tag queries in Async Controller as well.

Should some of those queries be on multiple lines? Like:
```sql
SELECT id 
FROM $tablename 
WHERE status = 'published' 
    and datedepublish <= :now 
    and datedepublish > '1900-01-01 00:00:01'
    and datechanged < datedepublish
```

### CODE REVIEW

**Comment by GawainLynch** [comment]
Database queries after my own heart! LGTM

**Comment by CarsonF** [comment]
Multiline yes/no?

**Comment by GawainLynch** [comment]
>Multiline yes/no?

Where it is long, absolutely!  [personal taste disclaimer]

**Comment by CarsonF** [comment]
They are all long for me....

**Comment by GawainLynch** [comment]
Historically, I've tended to use something like:

```
"SELECT slug, COUNT(slug) as count 
FROM $table 
WHERE taxonomytype = ? 
GROUP BY slug 
ORDER BY count DESC 
LIMIT ?"
```

But that can really piss some coders off

**Comment by CarsonF** [comment]
That's what I was thinking. Figured I would ask because of that point

File: src/Controllers/Async.php [hunk 1]
```diff
@@ -240,33 +240,26 @@ public function tags(Silex\Application $app, $taxonomytype)
...
         $table .= 'taxonomy';
 
-        $limit = $app['request']->get('limit', 20);
-
-        $query = sprintf(
-            'SELECT slug, COUNT(slug) as count from  %s where taxonomytype = ? GROUP BY  slug ORDER BY count DESC LIMIT %s',
-            $table,
-            intval($limit)
+        $results = $app['db']->fetchAll(
+            "SELECT slug, COUNT(slug) as count from $table where taxonomytype = ? GROUP BY slug ORDER BY count DESC LIMIT ?",
```
Comment (GawainLynch) [inline src/Controllers/Async.php]: s/from/FROM/
s/where/WHERE/

**Pull request merged**